checkov/terraform/plan_parser.py

```diff
@@ -144,20 +136,24 @@ def _prepare_resource_block(
     """
 
     resource_block: Dict[str, Dict[str, Any]] = {}
-    resource_block[resource["type"]] = {}
+    resource_type = resource["type"]
```

Comment:
Are we sure every resource will have `type` attribute?

Reply:
as far as I can tell, yes. Their docs don't really tell, what is a required field, but a resource/data block without a `type` would be new to me. If we find something like that, then we can fix it 🙂